ferrocene/doc/index/index.html

```diff
@@ -166,6 +166,23 @@ <h3>Compiler error codes</h3>
                     </a>
                 </div>
             </section>
+            <section>
+                <h2>Licensing and copyrights</h2>
+                <p class="note">
+                    These files contain licensing and copyright information for all files contained in the primary
+                    Rust project codebase, http://github.com/rust-lang/rust.
+                </p>
+                <div class="docs">
+                    <a href="COPYRIGHT.html">
+                        <h3>Rust codebase</h3>
+                        <p>Represents all files in the repository.</p>
+                    </a>
+                    <a href="COPYRIGHT-library.html">
+                        <h3>Rust standard libraries</h3>
+                        <p>Represents only the standard libraries (<b>core</b>, <b>alloc</b>, and <b>std</b>).</p>
```

Comment:
Couple of thoughts here (nitpicky, sorry)

- the way the links are currently it looks like you're linking to the thing, not to the license of the thing
- licences don't "represent" they apply to? 

So how about something like (in the same html structure you have now):

- [Rust license]() - applies to all files except the standard libraries below
- [Rust standard libraries license]() - applies to the standard libraries (<b>core</b>, <b>alloc</b>, and <b>std</b>

Reply:
Left similar feedback in https://github.com/ferrocene/ferrocene/pull/1395#discussion_r2034796737 too.

Reply:
@plaindocs am happy for all the nitpicks that propose improved reading experiences